plat/rockchip/rk3399/drivers/pmu/pmu.c

```diff
@@ -1261,15 +1261,15 @@ void cru_register_save(void)
 {
 	int i;
 
-	for (i = 0; i <= CRU_SDIO0_CON1; i = i + 4)
+	for (i = 0; i < CRU_SDIO0_CON1; i = i + 4)
```

Comment:
Is this really the right fix, though? I'm not super deep into this code, but AFAICT the point is to save/restore all CRU registers. With this patch you end up not saving the CRU_SDIO0_CON1 register. Looks like the real fix would be to increase the array size after all.

@dbasehore should also take a look.